src/Common/CosmosClient.test.ts

```diff
@@ -28,19 +28,6 @@ describe("tokenProvider", () => {
   afterEach(() => {
     jest.restoreAllMocks();
   });
-
```

Comment:
FYI, the `describe("tokenProvider", () => {})` doesn't have any tests anymore and should be removed or the tests should be fixed if the path is still used?